### ISSUE TITLE
-Wpointer-bool-conversion in net/core/ethtool.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#353]
```
net/core/ethtool.c:3023:19: warning: address of array 'ext_m_spec->h_dest' will always evaluate to 'true' [-Wpointer-bool-conversion]
                if (ext_m_spec->h_dest) {
                ~~  ~~~~~~~~~~~~^~~~~~
```

Patch sent: https://lore.kernel.org/lkml/20190208044652.32166-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git/commit/?id=8b34ec65b3d070c492cc6aca960097ae74d6143a

**Comment by nathanchance** [comment]
Fixed as of `next-20190211`: https://git.kernel.org/next/linux-next/c/8b34ec65b3d070c492cc6aca960097ae74d6143a
